src/thinsrv/CMakeLists.txt

```diff
@@ -201,6 +201,10 @@ if(INSTALL_DOC AND UNIX)
 	)
 endif()
 
+if(DP_PRECOMPILED_HEADERS)
+	target_precompile_headers(${srvlib} PRIVATE <QtCore> <QtNetwork> <QtSql> <libserver/jsonapi.h> <libserver/serverconfig.h>)
```

Comment:
Could you make these lists into multiple lines so that if/when they are changed it is less noisy in the patch? Thanks :-)